core/controller/src/main/scala/whisk/core/controller/Triggers.scala

```diff
@@ -205,13 +211,7 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
             }
           }
 
-          onComplete(saveTriggerActivation) {
```

Comment:
Might want to keep this code block around so the user knows if a failure occurred.

Reply:
They won’t here. No worse than today. The subsequent improvements will address this deficiency. You can see a sketch of the implementation here https://github.com/apache/incubator-openwhisk/compare/master...rabbah:trigger-activations?expand=1

Reply:
Then the api can’t be asynchronous wrt writing the record to the database.